appveyor.yml

```diff
@@ -7,6 +7,10 @@ environment:
   - nodejs_version: "8"
   - nodejs_version: "10"
 
+platform:
+  - cmd
```

Comment:
Is this a name you chose or something that AppVeyor has named?
If it's your choice: Maybe make it `use_cmd` to make this clearer.

Reply:
I think that can be whatever you want. If you prefer a prefix, I can add it, but I thought it's clear enough :D

Reply:
But, what about using `CommandPrompt` and `PowerShell`? Or do you prefer to have a verb in there?

Reply:
Does doesn't really matter, my confusion came from assuming this was a "magic string" you could set to trigger some platform - but it seems this is just selecting which part of the script is being executed.